### ISSUE TITLE
VAT from expense reports does not appear in VAT reports

### ISSUE THREAD

**Issue by rdoursenaud** [Dolibarr/dolibarr#4198]
The VAT on expense reports should appear in VAT reports.

**Comment by marcosgdf** [comment]
Should it appear? I don't think so…

**Comment by rdoursenaud** [comment]
Hum, I may have not explained this as thoroughly as needed.

Let me try again:
Expense reports allow using VAT.
If you want to be able recover that VAT from your expenses, this data should absolutely make its way to the VAT report.

I this helpful?

**Comment by marcosgdf** [comment]
Not from my point of view… but it really depends on your workflow. Adding non-deductible documents to VAT report will generate a wrong view of the VAT balance… 

What are expenses used for?

**Comment by rdoursenaud** [comment]
These expenses are deductible. They are meals, travels, etc. you pay for your staff !

**Comment by marcosgdf** [comment]
It depends. In Spain if you want to deduct VAT from invoices it must have the VAT from your company in the ticket/simplified invoice. In that case can be registered as a supplier invoice…

If it doesn't meet the requirements of our tax administration, then the VAT is considered as an expense and therefore non-deductible. 

It is very common (at least in Spain) not asking for an invoice of little amounts in a café, e.g.

**Comment by reeperbahnause** [comment]
As far as I know, in Germany the tax is deductible so it should be in the VAT report and in all other reports. Otherwise we need to create Invoices which makes the expense report module unnecessary.

**Comment by arunasr** [comment]
In UK, VAT is claimable on some expenses, as long as VAT receipt is provided.
If expenses cannot be included in the VAT report, what is the point of recording VAT in expense report?
In any case,
- Expense reports need to be extended to include optional company name and VAT reg number on each line (in case VAT is claimable),
- VAT report needs to have an option to include expense lines with VAT (depending on the country rules, or expense type).
  For example, business trip expenses, including meals and accommodation, are all VAT-claimable, but business entertainment (lunch meeting) is not.

**Comment by simnandez** [comment]
This issue is on an old version. It is very likely that this problem is fixed in the new versions. If no, please re-open a issue
